src/transport.ts

```diff
@@ -45,16 +46,31 @@ export const CERTHASH_CODE: number = 466
 // @TODO(ddimaria): seems like an unnessary abstraction, consider removing
 export interface WebRTCTransportComponents {
   peerId: PeerId
+  metrics?: Metrics
+}
+
+export interface WebRTCMetrics {
+  dialerEvents: CounterGroup
 }
 
 export class WebRTCTransport implements Transport {
   /**
    * The peer for this transport
    */
   private readonly components: WebRTCTransportComponents
+  private readonly metrics?: WebRTCMetrics
 
   constructor (components: WebRTCTransportComponents) {
     this.components = components
+
+    if (components.metrics != null) {
+      this.metrics = {
+        dialerEvents: components.metrics.registerCounterGroup('libp2p_webrtc_dialer_errors_total', {
```

Comment:
On the TCP version, `errors` is replaced with `events` in the 2 lines.  Are these errors?

Reply:
name has been changed